packages/design-system/src/styles/components/_Dialog.scss

```diff
@@ -54,7 +54,9 @@ $dialog-spacer: $spacer-4;
       background-image: url('#{$image-path}/close.svg');
     }
   } @else {
-    background-image: url('#{$image-path}/close--primary-darker.svg');
```

Comment:
Also needed to add :focus here, thanks for the catch @nichia

Reply:
i’m a bit confused why we even use the focus flag here, what was this originally? shouldn’t the the style here be to apply the darker close button on focus all the time

Reply:
- [Before the focus styles were implemented `close--primary-darker.svg` was used for focus](https://github.com/CMSgov/design-system/blame/89bf81730c3d60154045930ff7fc3679470baade/packages/core/src/components/Dialog/Dialog.scss#L50)
- [When the focus styles were first implemented `close.svg` was used for focus](https://github.com/CMSgov/design-system/blame/0f4386139d23724a999dfe60b6e34f3514c67965/packages/design-system/src/styles/components/_Dialog.scss#L53) because the image was black and fit with the rest of the black and gold focus styling.

Reply:
So I'm thinking we keep this `if/else` because of the color differences in the close buttons

Reply:
gotcha thanks for clarifying!